scripts/limonade/src/index.ts

```diff
@@ -138,6 +177,36 @@ async function run() {
     }
   };
 
+  const isOffMarket = (order: { state: Order; address: PublicKey }) => {
+    const priceInputMint = priceStore[order.state.inputMint.toString()];
+    const priceOutputMint = priceStore[order.state.outputMint.toString()];
+
+    if (!priceInputMint || !priceOutputMint) {
```

Comment:
i think you should check the price staleness here? otherwise, it's possible a price gets added but then doesn't update and then is stale later on but is still stored?

Reply:
done